tardis/io/tests/test_model_reader.py

```diff
@@ -393,6 +393,7 @@ def test_store_runner_to_hdf(simulation_verysimple, tmp_path):
         assert f["runner/use_gpu"][()] == runner_data["use_gpu"]
         assert np.array_equal(f["runner/v_inner"], runner_data["v_inner_cgs"])
         assert np.array_equal(f["runner/v_outer"], runner_data["v_outer_cgs"])
+        assert f["runner/nthreads"][()] == runner_data["nthreads"]
```

Comment:
Because this is a new line introduced by this PR, we will need to rename "runner" here to "transport" (as this line was non-existent when seen from ["renaming-PR"'s](https://github.com/tardis-sn/tardis/pull/2317) perspective.)

I suggest we merge this PR first, and then update ["the renaming PR"](https://github.com/tardis-sn/tardis/pull/2317) to rename this line as well, and then merge that.